[helm-operator] panic: odd number of arguments passed as key-value pairs for logging

## Bug Report

I've upgraded helm-operator image from 1.5.0 to 1.6.2 and it started crashing with the following panic:

```
========== Output of the operator container log =============

Flag --enable-leader-election has been deprecated, use --leader-elect instead.
2021-04-28T16:57:28.523Z	INFO	cmd	Version	{"Go Version": "go1.15.11", "GOOS": "linux", "GOARCH": "amd64", "helm-operator": "v1.6.1+git", "commit": "b131ca8ec77c96b9898470eba9560c30af0f23f3"}
2021-04-28T16:57:28.524Z	DPANIC	cmd	odd number of arguments passed as key-value pairs for logging	{"Namespace": "test-operator", "ignored key": "WATCH_NAMESPACE"}
github.com/go-logr/zapr.handleFields
	/go/pkg/mod/github.com/go-logr/zapr@v0.2.0/zapr.go:100
github.com/go-logr/zapr.(*zapLogger).Info
	/go/pkg/mod/github.com/go-logr/zapr@v0.2.0/zapr.go:127
github.com/operator-framework/operator-sdk/internal/cmd/helm-operator/run.run
	/workspace/internal/cmd/helm-operator/run/cmd.go:138
github.com/operator-framework/operator-sdk/internal/cmd/helm-operator/run.NewCmd.func1
	/workspace/internal/cmd/helm-operator/run/cmd.go:69
github.com/spf13/cobra.(*Command).execute
	/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:854
github.com/spf13/cobra.(*Command).ExecuteC
	/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:958
github.com/spf13/cobra.(*Command).Execute
	/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:895
main.main
	/workspace/cmd/helm-operator/main.go:40
runtime.main
	/usr/local/go/src/runtime/proc.go:204
panic: odd number of arguments passed as key-value pairs for logging

goroutine 1 [running]:
go.uber.org/zap/zapcore.(*CheckedEntry).Write(0xc0000d4b00, 0xc0003d15c0, 0x1, 0x1)
	/go/pkg/mod/go.uber.org/zap@v1.15.0/zapcore/entry.go:230 +0x55f
go.uber.org/zap.(*Logger).DPanic(0xc0005118c0, 0x1d8ce01, 0x3d, 0xc0003d15c0, 0x1, 0x1)
	/go/pkg/mod/go.uber.org/zap@v1.15.0/logger.go:215 +0x85
github.com/go-logr/zapr.handleFields(0xc0005118c0, 0xc00063f0c0, 0x1, 0x1, 0x0, 0x0, 0x0, 0x10, 0x1a62300, 0x1b60701)
	/go/pkg/mod/github.com/go-logr/zapr@v0.2.0/zapr.go:100 +0x5e5
github.com/go-logr/zapr.(*zapLogger).Info(0xc00063f0b0, 0x1d525ef, 0x22, 0xc00063f0c0, 0x1, 0x1)
	/go/pkg/mod/github.com/go-logr/zapr@v0.2.0/zapr.go:127 +0xb0
github.com/operator-framework/operator-sdk/internal/cmd/helm-operator/run.run(0xc0004c7b80, 0xc00019b000)
	/workspace/internal/cmd/helm-operator/run/cmd.go:138 +0xb25
github.com/operator-framework/operator-sdk/internal/cmd/helm-operator/run.NewCmd.func1(0xc0004c7b80, 0xc0003d10c0, 0x0, 0x4)
	/workspace/internal/cmd/helm-operator/run/cmd.go:69 +0x9d
github.com/spf13/cobra.(*Command).execute(0xc0004c7b80, 0xc0003d1080, 0x4, 0x4, 0xc0004c7b80, 0xc0003d1080)
	/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:854 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0xc0004c78c0, 0xc0007dff60, 0x1, 0x1)
	/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:895
main.main()
	/workspace/cmd/helm-operator/main.go:40 +0xe5
```

#### Environment

**Operator type:**

/language helm

**Kubernetes cluster type:**

I am using `quay.io/operator-framework/helm-operator:v1.6.2` in RedHat certification environment, the issue occurs during bundle deployment tests.

--- a/changelog/fragments/fmt-info-logs.yaml
+++ b/changelog/fragments/fmt-info-logs.yaml
@@ -1,0 +1,4 @@
+entries:
+  - description: >
+      Format ansible-operator and helm-operator `run` command logs correctly.
+    kind: bugfix

--- a/internal/cmd/ansible-operator/run/cmd.go
+++ b/internal/cmd/ansible-operator/run/cmd.go
@@ -139,7 +139,7 @@ func run(cmd *cobra.Command, f *flags.Flags) {
 	namespace, found := os.LookupEnv(k8sutil.WatchNamespaceEnvVar)
 	log = log.WithValues("Namespace", namespace)
 	if found {
-		log.V(1).Info("Setting namespace with value in %s", k8sutil.WatchNamespaceEnvVar)
+		log.V(1).Info(fmt.Sprintf("Setting namespace with value in %s", k8sutil.WatchNamespaceEnvVar))
 		if namespace == metav1.NamespaceAll {
 			log.Info("Watching all namespaces.")
 			options.Namespace = metav1.NamespaceAll

--- a/internal/cmd/helm-operator/run/cmd.go
+++ b/internal/cmd/helm-operator/run/cmd.go
@@ -135,7 +135,7 @@ func run(cmd *cobra.Command, f *flags.Flags) {
 	namespace, found := os.LookupEnv(k8sutil.WatchNamespaceEnvVar)
 	log = log.WithValues("Namespace", namespace)
 	if found {
-		log.V(1).Info("Setting namespace with value in %s", k8sutil.WatchNamespaceEnvVar)
+		log.V(1).Info(fmt.Sprintf("Setting namespace with value in %s", k8sutil.WatchNamespaceEnvVar))
 		if namespace == metav1.NamespaceAll {
 			log.Info("Watching all namespaces.")
 			options.Namespace = metav1.NamespaceAll

